CMakeLists.txt

```diff
@@ -16,24 +16,85 @@ project(
     LANGUAGES CXX
 )
 
-#---- C++ standard that we need (at least).
-set(CMAKE_CXX_STANDARD          20   CACHE BOOL "" FORCE)
-set(CMAKE_CXX_STANDARD_REQUIRED True CACHE BOOL "" FORCE)
+#---- Options.
+option(KOKKOSUTILS_ENABLE_TESTS "Enable testing" ON)
+option(KOKKOSUTILS_ENABLE_DOC   "Enable documentation" ON)
 
-#---- Find Kokkos.
-find_package(
-    Kokkos
-    ${KOKKOS_VERSION}
-    REQUIRED
+#---- Global property that helps us keep track of files that will automatically be added
+#     to our Doxygen documentation.
+define_property(GLOBAL PROPERTY FILES_FOR_DOC 
+    BRIEF_DOCS "Files that will be added to the Doxygen documentation."
 )
 
+#---- Find Kokkos.
+#
+# Currently, we require the Kokkos::kokkoscore target. Other targets will be added as needed.
+if(NOT TARGET Kokkos::kokkoscore OR NOT T)
```

Comment:
```suggestion
if(NOT TARGET Kokkos::kokkoscore)
```

Reply:
...?

Reply:
I think it's because both conditions pass 😄 But I'll clean it up :sparkles: